setup.py

```diff
@@ -59,13 +59,13 @@ def read(fname):
         'colorama>=0.3,<0.4',
         'hgapi>=1.7,<2',
         'Jinja2>=2.7.0,<3',
-        'cryptography>=2.1.4,<2.2',
+        'cryptography>=2.1.4,<=2.3',
         'PyJWT>=1.0,<2.0',
         'pathlib>=1.0.1,<1.1',
         'jsonschema>=2.4.0,<3.0',
         'argcomplete>=0.8.0,<2.0',
         'mbed_test_wrapper>=0.0.3,<2.0.0',
-        'valinor>=0.0.0,<1.0',
+        'valinor>=1.1.4,<2.0',
```

Comment:
Valinor is only as [release 1.1.3](https://pypi.org/project/valinor/), should this reflect that?

Reply:
I opened [another PR there](https://github.com/ARMmbed/valinor/issues/32) last night because PyOCD updated their dependencies and that broke it again. This keeps happening because PyOCD [is not considered stable yet](https://github.com/mbedmicro/pyOCD#status), which gives them the freedom of breaking things.

With this in mind, maybe Valinor should do a hard bind to a specific known-good version of PyOCD instead. If Valinor and Yotta go into active development again, then it will make sense to use the latest versions of dependencies.

Reply:
I see, @0xc0170 please also merge/release https://github.com/ARMmbed/valinor/issues/32.

@pandasauce, I can see how this is annoying. Feel free to suggest a hard bind version, that sounds sensible.

Reply:
> I see, @0xc0170 please also merge/release ARMmbed/valinor#32.

done